src/CLStrings.ts

```diff
@@ -8,7 +8,8 @@ export enum CLStrings {
     MEMORY_MANAGER_VALUE_LIST_EXCEPTION = "Entity is multi-value. Use AS_VALUE_LIST",
     MEMORY_MANAGER_NUMBER_LIST_EXCEPTION = "Entity is multi-value. Use AS_NUMBER_LIST",
     MEMORY_MANAGER_BOOLEAN_LIST_EXCEPTION = "Entity is multi-value. Use AS_BOOLEAN_LIST",
-    MEMORY_MANAGER_INVALID_ENUM_EXCEPTION = "Entity is an ENUM. Provided value must be one of the enum values",
+    MEMORY_MANAGER_INVALID_ENUM_EXCEPTION1 = " is an ENUM Entity. Provided value (",
+    MEMORY_MANAGER_INVALID_ENUM_EXCEPTION2 = ") is not one of the enum values of this Entity: ",
```

Comment:
You can pass values into the internationalized strings. 

something like:
`{entityName} is an ENUM Entity. Provided value ({ value }) is not one of the enum values of this Entity: { possibleValues }`

It could prevent you from having to keep breaking up the internationalization strings into new variables arbitrarily based on how you want to output it.

Reply:
It isn't an internationalized string, but used in a throw